index.html

```diff
@@ -20,27 +20,27 @@
     </nav>
     <div class="headline">
     <div class="description">
-      <h1 class="name"> I am Aime
-          Glad to see you!</h1>
+      <h2 class="name"> I am Aime
+          Glad to see you!</h2>
          
       <p class="text"> I am a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
            </p>
       </div>
       <div class="contact">
         <button>
 
-          <h2 class="media">LET'S CONNECT:</h2>
+          <h2>LET'S CONNECT</h2>
         
         </button>
         <ul>
          
-          <li><a href="https://twitter.com/AimeSebagabo"><i class='bx bxl-twitter'></i></a></li>
+          <li><a href="#"><i class='bx bxl-twitter'></i></a></li>
 
-          <li><a href="https://www.linkedin.com/in/jean-aime-sebagabo-279aa2207/"></a><i class='bx bxl-linkedin'></i></a></li>
+          <li><a href="#"></a><i class='bx bxl-linkedin'></i></a></li>
       
-           <li><a href="http://yourgmailhandle"><i class='bx bxl-gmail'></i></a></li>
+           <li><a href="#"><i class='bx bxl-gmail'></i></a></li>
       
-           <li><a href="https://github.com/Hazard10-eng"><i class='bx bxl-github'></i></a</a</li>
+           <li><a href="#"><i class='bx bxl-github'></i></a</a</li>
         </ul>
```

Comment:
- [ ] Good job implementing the changes suggested, however you only made changes to the let's connect part, please make sure the social icons also match the figma design, as one of the goal of this project is to learn to parse a figma template as close as possible, currently your social icons are like this :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206864500-5409b887-b80f-4580-ba4d-234259d087f9.png)

how it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206864541-876853a7-721d-4250-bb60-af022b9eb915.png)